sunpy/io/tests/test_fits.py

```diff
@@ -96,3 +98,13 @@ def test_extra_comment_write(tmpdir):
     outfile = tmpdir / "test.fits"
     sunpy.io.fits.write(str(outfile), data, header)
     assert outfile.exists()
+
+
+def test_simple_write_compressed(tmpdir):
+    data, header = sunpy.io.fits.read(AIA_171_IMAGE)[0]
+    outfile = os.path.join(tmpdir, "test.fits")
```

Comment:
And I made you to break your code!!!
What's `tmpdir`?? why it overloads `/` as concatenation of paths?
```suggestion
    outfile = tmpdir / "test.fits"
```
Sorry for breaking your test!